lib/modules/manager/terraform/extract.spec.ts

```diff
@@ -104,6 +105,13 @@ describe('modules/manager/terraform/extract', () => {
       expect(res).toMatchSnapshot();
     });
 
+    it('extracts kubernetes resources', async () => {
+      const res = await extractPackageFile(kubernetes, 'kubernetes.tf', {});
+      expect(res.deps).toHaveLength(16);
+      expect(res.deps.filter((dep) => dep.skipReason)).toHaveLength(2);
+      expect(res).toMatchSnapshot();
```

Comment:
Please use `toMatchObject` and only match required fields

Reply:
Done